components/ContactDetailsAskModal.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <b-modal
+    id="contact-details-ask-modal"
+    v-model="showModal"
+    scrollable
+    size="lg"
+    no-stacking
+    no-fade
+  >
+    <div class="d-flex justify-content-between flex-wrap">
+      <b-form-group
+        class="flex-grow-1 nobot"
+        label="Your postcode:"
+        :label-for="'replytomessage-' + 'message.id'"
+        description=""
+      >
+        <PostCode @selected="savePostcode" />
+        <div class="text-muted text--small mt-1">
+          <v-icon icon="lock" /> Other freeglers won't see this.
+        </div>
+      </b-form-group>
+      <SettingsPhone
+        v-if="me"
+        label="Your mobile:"
+        size="lg"
+        hide-remove
+        auto-save
+        input-class="phone"
+      />
+    </div>
+    <p class="text-muted">
+      <b-button
+        v-if="!showWhyAsk"
+        size="sm"
+        variant="link"
+        @click="showWhyAsk = true"
+      >
+        Why do we ask for this?
```

Comment:
This hide/show stuff was to avoid too much info on the screen.  We don't have that problem now this is in a modal.  So please:

- remove that function so that we always show the "We ask for..." text
- Move that text up to the top so that it acts as an introduction.
- Put the last sentence into a new paragraph
- Use `<strong>` on "We won't show either of these".